build-system/tasks/visual-diff.js

```diff
@@ -342,27 +343,29 @@ async function runVisualTests(page, visualTestsConfig) {
 /**
  * Cleans up any existing AMP config from the runtime and 3p frame.
  */
-function cleanupAmpConfig() {
+async function cleanupAmpConfig() {
   log('verbose', 'Cleaning up existing AMP config');
-  AMP_RUNTIME_TARGET_FILES.forEach(targetFile => {
-    execOrDie(
-        `gulp prepend-global --local_dev --target ${targetFile} --remove`,
-        {'stdio': 'ignore'});
-  });
+  for (const targetFile of AMP_RUNTIME_TARGET_FILES) {
```

Comment:
There was a recent change to build a non-polyfill version of the runtime at `dist/amp-esm.js`. I'm not sure which of the visual tests use it, but we should definitely add it to `AMP_RUNTIME_TARGET_FILES`.

Reply:
Done